### PR TITLE
Assigned-to filtering, task filters in workflow

### PR DESCRIPTION
**Pull request by longhotsummer**
Review automatically linked publication documents – Laws Africa 2019-04-16 14-01-24

Tasks – South Africa – Laws Africa 2019-04-16 14-01-52

### CODE REVIEW

**Comment by longhotsummer** [comment]
Fixes #386

**Comment by goose-life** [comment]
looks awesome!

- the `x` next to 'Assigned to' in the dropdown: I expected this to clear who it's assigned to, not just close the dropdown?
Screenshot from 2019-04-16 21-59-35

- this is pretty minor, but in the workflow detail view when there's nothing assigned to that user the default message makes less sense that the 'No tasks found.` of the task list view:
Screenshot from 2019-04-16 22-01-52

**Comment by longhotsummer** [comment]
I removed the dropdown heading, I agree it was weird. We don't have a huge amount of control over that functionality, so I figured no heading is easier.

Review automatically linked publication documents – Laws Africa 2019-04-17 09-47-13

File: indigo_app/views/workflows.py [hunk 1]
```diff
@@ -51,11 +51,29 @@ class WorkflowDetailView(WorkflowViewBase, DetailView):
...
+
     def get_context_data(self, *args, **kwargs):
         context = super(WorkflowDetailView, self).get_context_data(**kwargs)
 
-        tasks = self.object.tasks.all()
-        context['has_tasks'] = bool(tasks)
+        context['form'] = self.form
+        tasks = self.form.filter_queryset(self.object.tasks).all()
+
+        context['has_tasks'] = self.object.tasks.count() > 1
```
Comment (goose-life) [inline indigo_app/views/workflows.py]:
```suggestion
        context['has_tasks'] = self.object.tasks.count() > 0
```

Comment (longhotsummer) [reply indigo_app/views/workflows.py]: LOL!

**Pull request merged**